themes/hugo-rladies/layouts/partials/funcs/social_media.html

```diff
@@ -2,60 +2,56 @@
 
 {{ if eq .type "twitter" }}
   {{ $data.Set "string" "https://twitter.com/%s" }}
-  {{ $data.Set "fa" "fab fa-twitter" }}
-  
+  {{ $data.Set "fa" "fa-brands fa-twitter" }}
```

Comment:
btw should we remove twitter links

Reply:
I'm note sure. There are many of them in the directory, and while WE dont use it, I'm not sure we should not link to them if other do? I'm a little torn.